README.md

```diff
@@ -1,9 +1,14 @@
 # eredis (Nordix fork)
 
+[![Hex pm](http://img.shields.io/hexpm/v/eredis.svg?style=flat)](https://hex.pm/packages/eredis)
+
 Non-blocking Redis client with focus on performance and robustness.
 
-Improvements and changes in this fork compared to `wooga/eredis`: TLS support
-and error handling corrections. See [CHANGELOG.md](CHANGELOG.md).
+Improvements and changes in this fork compared to `wooga/eredis`:
+
+* TLS support
+* Error handling corrections.
+See [CHANGELOG.md](CHANGELOG.md).
```

Comment:
This looks like an indentation error. Is "See CHANGELOG" supposed to be included in the last bullet? If yes, indent it properly. It seems to work already though: https://github.com/Nordix/eredis/blob/0444273ab1493ecadb388e8c0af6d36df90645cf/README.md

Even if error handling is the only relevant difference compared to wooga/eredis now, I don't think it will remain that way. Better mention the CHANGELOG file in a generic way.

Idea: Remove the bullet list and just say something like "Improvements and changes in this fork compared to `wooga/eredis` including TLS support and TCP error handling corrections. See CHANGELOG.md for details."